packages/react/src/components/AnimatedHeader/components/Tiles/BaseTile/BaseTile.tsx

```diff
@@ -17,19 +17,13 @@ import { GlassTile, type GlassTileProps } from '../GlassTile/GlassTile';
 /** Base Tile router */
 export type TileVariant = 'glass' | 'aiPrompt' | 'ai';
 
-export type BaseTileProps =
-  | ({ variant?: 'glass' } & GlassTileProps)
-  | ({ variant: 'aiPrompt' } & AIPromptTileProps)
-  | (({ variant: 'ai' } & AITileProps) & {
-      id?: string;
-      tileId?: string | null;
-    });
+export type BaseTileProps = GlassTileProps | AIPromptTileProps | AITileProps;
 
 function inferVariant(props: BaseTileProps): TileVariant {
   if (props.variant) {
     return props.variant;
   }
-  if (props.id === 'ai-tile') {
+  if (props.tileId === 'ai-tile') {
```

Comment:
This is designed as a fallback before the variant prop was introduced. Before variant prop was introduced every Tile was a Glass Tile unless the Tile `id` was set to 'ai-tile' which wasn't ideal.. so the variant prop was added. So this needs to stay as just `id` and not `tildId` as a fallback for users who aren't using `variant` yet